core/templates/dev/head/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -40,6 +40,9 @@ oppia.controller('SettingsTab', [
       EXPLORATION_TITLE_INPUT_FOCUS_LABEL);
 
     $scope.CATEGORY_LIST_FOR_SELECT2 = [];
+    //Empty category added to force dropdown to scroll top
```

Comment:
Use three backticks for multiline code: https://guides.github.com/features/mastering-markdown/

@souravbadami, any thoughts? I don't really mind either way.